src/svg/ViewWireframe.ts

```diff
@@ -0,0 +1,49 @@
+import { ViewStyle } from "react-native"
+import * as yoga from "yoga-layout"
+
+import { styleFromComponent } from "../component-to-node"
+import { RenderedComponent } from "../index"
+import wsp from "../whitespace"
+import {
+  dashStyles,
+  filledPathForSide,
+  getScaledBorderRadius,
+  pathForRect,
+  scaleSides,
+  Sides,
+  sidesEqual,
+  strokedPathForSide,
+} from "./borders"
+import { $ } from "./util"
+
+export default ({ top, left, width, height }: yoga.Layout, style: any) => {
+  const borderRadii = getScaledBorderRadius(style, width, height)
+  const borderWidths: Sides<number> = [1, 1, 1, 1]
+  const borderWidth = 1
+
+  const attributes: any = {
+    "type": "View",
+    "fill": "none",
+    "stroke": "black",
+    "stroke-width": borderWidth,
+  }
+
+  if (sidesEqual(borderRadii)) {
+    const borderRadius = borderRadii[0]
+    // Offset size by half border radius, as RN draws border inside, whereas SVG draws on both sides
+    return $("rect", {
```

Comment:
@orta It's not hooked up yet, but does this look like it would work?

Reply:
Yep 👍